scale-test/fioload-template.yaml

```diff
@@ -0,0 +1,47 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: @podname@
+  labels:
+    app: @testid@
+spec:
+  restartPolicy: Never
+  @nodeSelector@:
+    type: worker
+  containers:
+  - command:
+    - sh
+    - -c
+    - 'fio --blocksize=@bs@ --verify=0 --randrepeat=0 --filename=/mnt/file1 --ioengine=libaio --readwrite=randread --runtime=60s --time_based --ramp_time=2s --size=3G --name=test --numjobs=8 --direct=1 --iodepth=2 --group_reporting > /tmp/fiotest; exit 0;'
```

Comment:
Fio has an option to specify the output file, I would use that as it is also not needed to collect the interval updates that fio produces.